azurerm/helpers/authentication/validation.go

```diff
@@ -49,3 +49,22 @@ func (c *Config) ValidateServicePrincipal() error {
 
 	return err.ErrorOrNil()
 }
+
+func (c *Config) ValidateMsi() error {
+	var err *multierror.Error
+
+	if c.SubscriptionID == "" {
```

Comment:
How can we get the default subscription info here like populateSubscriptionFromCLIProfile in MSI ?